src/plays/react-todo-app/react-todo.css

```diff
@@ -0,0 +1,142 @@
+.todo-play-body {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    background-image: linear-gradient(0deg, rgba(34, 193, 195, 1) 0%, rgba(253, 187, 45, 1) 100%);
+    color: white;
+    font-family: "Poppins", sans-serif;
+    height: 100%;
```

Comment:
I think, better to remove the height altogether.

It will be better as in when you add too many TODOs, the background will appear properly and the scroll only on the body.

Reply:
Ok